AXX-crl-enhancements.md

```diff
@@ -0,0 +1,220 @@
+AXX: Certificate Revocation List Enhancements
+----
+* Author(s): Gregory Cooke (@gtcooke94, @gregorycooke)
+* Approver: TODO 
+* Status: TODO
+* Implemented in: C, Go
+* Last updated: 2023-07-25
+* Discussion at: TODO
+
+## Abstract
+
+gRPC has an MVP implementation for CRLs (Certificate Revocation Lists). We wish to provide a more generic interface. In addition, we will provide two concrete implementations of this interface similar to Credential Reloading - a static provider and a directory reloader.
```

Comment:
I'd say 'more generic and flexible to enable developers to implement desired business functionality fitting their landscape'

Reply:
Let's also add a link to Credentials Reloading docs/gRFCs

Reply:
Changed - link is already in the Related Proposals section